queries/c.scm

```diff
@@ -182,7 +182,7 @@
 ) @_.domain
 
 ;;!! aaa = 0;
-(
+(_
```

Comment:
We need a common ancestor for all captures. Nodes not intersecting with the search range will not be matched.

Reply:
For more information
https://github.com/cursorless-dev/cursorless/issues/1769#issuecomment-2582673327